packages/client/src/hooks/useNotifications.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  nextNotification,
+  evictNotification,
+} from "@/store/actions/notificationsActions";
+import { getActiveNotification } from "@/store/selectors/notifications";
+import { NotificationInterface } from "@/types/store";
+import { useEffect, useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+interface useNotificationsParams {
```

Comment:
It is a convention to write the types, interfaces and enums in **PascalCase**. This is to prevent from clashing with JS objects maybe named the same way, only in **camelCase**